### DISCUSSION TITLE
Error in console.

### DISCUSSION THREAD

**Discussion by Neu3BecTHo** [Pryaxis/TShock#2603] [Q&A]
Hello. I am using the latest release of the release and I see this error when writing in the chat.
![image](https://user-images.githubusercontent.com/87427271/159158371-ed9d6576-0f2f-4cce-9a37-bdfee2ed960d.png)

**Comment by Arthri** [comment]
Hello! Thanks for reporting to the issue. I believe the error is in the config. Have you modified the config? And if so what is the value of these two options:
```json
    "ChatFormat": "{1}{2}{3}: {4}",
    "ChatAboveHeadsFormat": "{2}",
```
